packages/react-native-web/src/exports/CheckBox/index.js

```diff
@@ -11,12 +11,12 @@
  */
 
 import applyNativeMethods from '../../modules/applyNativeMethods';
```

Comment:
This is the only internal dependency for `CheckBox`. In user-land, a `View` ref could be used to add these methods instead.